Okta/OktaOidc/OktaOidc.swift

```diff
@@ -15,7 +15,7 @@ import UIKit
 public class OktaOidc: NSObject {
 
     // Current version of the SDK
-    @objc public static let VERSION = "3.1.0"
+    @objc public static let VERSION = "3.2.0"
```

Comment:
This is not required anymore, as framework version should be now in plist file. Please remove

Reply:
Done.